resources/js/Pages/Employee/Partials/Hierarchy.vue

```diff
@@ -309,11 +309,17 @@ export default {
   },
 
   watch: {
-    managersOfEmployee(value) {
-      this.localManagersOfEmployee = value;
+    managersOfEmployee: {
```

Comment:
This feature is broken. Works 50% of the time. Also, design seems broken when searching an employee - the Dots symbol that represents a loading event, seems completely off here.

Reply:
This is working fine.